packages/measure/src/server/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const PORT = 3000;
+export const WEBAPP_URL =
+  process.env.API_URL === "local"
```

Comment:
localhost in both cases => choose better naming?

Reply:
Changed it for `DEVELOPMENT_MODE`